config.json

```diff
@@ -15,7 +15,8 @@
       "unlocked_by": null,
       "difficulty": 1,
       "topics": [
-
+        "text formatting",
+        "optional values"
```

Comment:
I feel a bit stupid for asking, but are there any _options_ as such in [`hello-world`](https://github.com/exercism/r/blob/master/exercises/hello-world/example.R)? Maybe me being not a native English speaker is the cause for this question. I would use only the 1st topic, plus "strings".